### DISCUSSION TITLE
Support JSX component exports in design systems (@my/ui)

### DISCUSSION THREAD

**Discussion by ALegendsTale** [tamagui/tamagui#272] [Ideas]
Currently exporting a JSX component inside of a design system exhibits `ERROR: The JSX syntax extension is not currently enabled.` Only `styled` exports are allowed as briefly noted here in the [docs](https://tamagui.dev/docs/guides/design-systems#configuration-notes). I'd like to place non-tamagui components inside of my design system and make it a catch-all for ui components. Could this be supported?

**Comment by natew** [comment]
This should work now.
